tests/e2e/test_rosacli_node_pool.go

```diff
@@ -730,4 +730,78 @@ var _ = Describe("Edit nodepool",
 				analyzeUpgrade(nodePoolManualName, "manual")
 				analyzeUpgrade(nodePoolAutoName, "automatic")
 			})
+
+		It("create/edit nodepool with node_drain_grace_period to HCP cluster via ROSA cli can work well - [id:72715]",
+			labels.High, labels.NonClassicCluster,
+			func() {
+				By("check help message for create/edit machinepool")
+				help, err := machinePoolService.RetrieveHelpForCreate()
+				Expect(err).ToNot(HaveOccurred())
+				Expect(help.String()).To(ContainSubstring("--node-drain-grace-period"))
+				help, err = machinePoolService.RetrieveHelpForEdit()
+				Expect(err).ToNot(HaveOccurred())
+				Expect(help.String()).To(ContainSubstring("--node-drain-grace-period"))
+
+				nodeDrainGracePeriodsReqAndRes := []map[string]string{{"20": "20 minutes", "20 hours": "1200 minutes", "20 minutes": "20 minutes"}}
```

Comment:
Here needs a step with clear description,rather bellow By("check help message for create/edit machinepool") as below code is not testing the help message